apps/rest/run_evadb.py

```diff
@@ -0,0 +1,57 @@
+from flask import Flask,flash, request, redirect,render_template
+from flask_restful import Api, Resource
+import os
+from werkzeug.utils import secure_filename
+import evadb
+from pathlib import Path
+
+cursor = evadb.connect().cursor()
+cwd = os.getcwd()
+UPLOAD_FOLDER = cwd + "/files"
+ALLOWED_EXTENSIONS = {'txt', 'pdf', 'png', 'jpg', 'jpeg', 'gif', 'mp3'}
+
+app = Flask(__name__)
+app.config['UPLOAD_FOLDER'] = UPLOAD_FOLDER
+api = Api(app)
+app.secret_key = "evadb-rest-apis"
+
+# Get Query response
+@app.route('/query/<query>',methods = ['GET'])
```

Comment:
SQL queries usually have space. I am wondering how this will work. Does the user need to escape the url?

Reply:
No. I tried with spaces and it automatically converts it to spaces so it should be fine

Reply:
Also I added an input box similar to upload. It's easier to enter text that way